Automattic/wp-calypso
# Stats: Adding Subscribers stats page

Related to #75207

## Proposed Changes

* create a new Stats page and a tab in the navigation for `Subscribers`
* put the page behind `stats/subscribers-section` feature flag
* move the existing Subscribers components to the new page

## Testing Instructions



* Open the live branch for this PR
* Navigate to the traffic stats page at `/stats/day/:siteSlug` 
* Verify that the `Subscribers` tab is not visible next to the `Insights` tab
* Add the query string `?flags=stats/subscribers-section` to enable the subscriber counts page
* Verify that the `Subscribers` tab is visible next to the `Insights` tab
* Navigate to the page and verify that it loads with a line chart

![SCR-20230405-b0t](https://user-images.githubusercontent.com/112354940/229905721-0ae0498e-a38d-4ec1-8286-280246437c07.png)


## Pre-merge Checklist
